packages/transformer/src/IModelImporter.ts

```diff
@@ -257,7 +258,14 @@ export class IModelImporter {
       ) {
         this.onUpdateElement(elementProps);
       } else {
-        this.onInsertElement(elementProps);
+        const doesElementExistInTarget = this.targetDb.elements.tryGetElement(
```

Comment:
We definitely need a way to check this.. but I sort of want to avoid calling tryGetElement everytime. I will comment in imodeltransformer.ts for my suggestions: See comment on "targetElementProps.id = sourceElement.id"

Reply:
added a new option `fromTransformation` that will be used to indicate whether importer class comes from transformation. If it is false, we can call tryGetElement to see whether an element exists in target. Otherwise, if importer comes from transformation, we can use the set to keep track of the elements to update during transformation